app/controllers/links_controller.rb

```diff
@@ -20,13 +21,28 @@ def index
     end
   end
 
+  def sort_links
+    case params[:sort_by]
+    when 'Added On'
+      @sorted_links = params[:search_string].empty? ? current_user_links.order_by_created_at.paginate(page: page) : search_list.reorder('created_at DESC').paginate(page: page)
```

Comment:
`params[:search_string]` appears multiple time. So it is candidate for local variable.